Makefile

```diff
@@ -187,6 +187,9 @@ KUTTL ?= $(LOCALBIN)/kubectl-kuttl
 export KUBE_VERSION ?= 1.25
 KIND_CONFIG ?= kind-$(KUBE_VERSION).yaml
 
+# Default namespace for minio install
+MINIO_NAMESPACE ?= default
```

Comment:
We could change this and install minio in a different namespace. The default namespace does not work on OCP. The minio manifests should create namespace where the instance will be created.

Reply:
I agree. Probably the `storage` namespace is a good one?

Reply:
or just `minio` to be idiomatic.